src/pages/Account/types.ts

```diff
@@ -8,3 +8,8 @@ export interface IUser {
 export interface ILogout {
   refreshToken: string;
 }
+
+export interface IAccountInfo {
```

Comment:
Is there any sense to have the empty object as the account info?

Reply:
I changed that it can't be empty now.